src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class EditorHandler : AbstractViewHandler<IEditor, UITextView>
+	{
+		static readonly int BaseHeight = 30;
```

Comment:
This is to match the fact that Android has the same size view and also seems Forms used the same height as well.